File: app/src/main/kotlin/com/krsk/qiitareader/domain/repository/ItemRepository.java
package com.krsk.qiitareader.domain.repository;

import com.krsk.qiitareader.domain.model.entity.Item;

import java.util.List;

import rx.Observable;

/**
 * Created by bl-lia on 2/1/16.
 */
public interface ItemRepository {

    Observable<List<Item>> getItems();
}

File: app/src/main/kotlin/com/krsk/qiitareader/domain/repository/ItemRepository.kt
package com.krsk.qiitareader.domain.repository

import com.krsk.qiitareader.domain.model.entity.Item

import rx.Observable

/**
 * Created by bl-lia on 2/1/16.
 */
interface ItemRepository {

    fun getItems(): Observable<List<Item>>
}
